Release_Protocol.md

```diff
@@ -157,6 +169,8 @@ description:
 
 Click "Publish release".
 
+Note: It will take ReadTheDocs a few minutes to sync with the new release and build the new stable and versioned rendering. 
```

Comment:
I feel like this doesn't really serve much purpose here. Either they'll build without intervention or they won't. You can just say "Verify that they do", at which point you might as well point them to the right page, e.g.,

```suggestion
Verify ReadTheDocs builds complete and publish. If needed, manually
trigger [builds](https://readthedocs.org/projects/bids-specification/builds/)
for `stable` and the most recent tag.
```